src/main/java/com/pnudev/springonlinedelivery/service/OrderServiceImpl.java

```diff
@@ -0,0 +1,15 @@
+package com.pnudev.springonlinedelivery.service;
+
+import com.pnudev.springonlinedelivery.models.Order;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService{
+    @Override
+    public List<Order> getOrders() {
+        return Collections.emptyList();
```

Comment:
Create a //TODO for unimplemented features

Reply:
Done